modules/gitops-creating-an-application-by-using-the-argo-cd-dashboard.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+// * depoying-an-application-with-argo-cd.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "deploying-a-spring-boot-application-with-argo-cd"]
+:app:
+endif::[]
+
+[id="creating-an-application-by-using-the-argo-cd-dashboard_{context}"]
+= Creating an application by using the Argo CD dashboard
+
+Argo CD provides a dashboard which allows you to create applications.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the _cluster_ directory to the `cluster-configs` application. The directory defines OpenShift web console cluster configurations that add a link to the *Red Hat Developer Blog* under the {rh-app-icon} menu and define a namespace `spring-petclinic` on the cluster.
```

Comment:
_cluster_/`cluster`
under the {rh-app-icon} menu in the web console
menu and define a/menu, and a